samples/apps/copilot-chat-app/webapi/Skills/ChatSkill.cs

```diff
@@ -394,7 +394,12 @@ private bool TryExtractJsonFromPlanResult(string openApiSkillResponse, out strin
         catch (JsonException)
         {
             // Expected if not valid JSON.
-            this._logger.LogDebug("Unable to extract JSON from planner response, it is likely not JSON formatted");
+            this._logger.LogDebug("Unable to extract JSON from planner response, it is likely not from an OpenAPI skill.");
+        }
+        catch (InvalidOperationException)
+        {
+            // Expected if valid JSON but not from an OpenAPI-based skill.
+            this._logger.LogDebug("Unable to extract JSON from planner response, it is likely already proper JSON.");
```

Comment:
If it's already proper json, shouldn't you return the openApiSkillResponse instead of string.Empty?

Reply:
I'll update the comment. We don't technically know if it's JSON, we just know we didn't get the format we expected. We want to inform the caller that we failed to verify the output and that our out parameter should not be used.